### PR TITLE
TRT-1718: Add new intervals for kubelet metrics endpoints down

### PR DESCRIPTION
**Pull request by dgoodwin**
New monitortest for querying prom at the end of run to get precise times
it found the kubelet /metrics and /metrics/cadvisor endpoints down. This
is to attempt to gain better insights into what's going on in the
cluster at the time this occurs.

Also includes a test that will flake if we see this outside a node
reboot.

### CODE REVIEW

**Comment by dgoodwin** [comment]
The problem we're chasing here *might* only be happening in 4.16, I've been testing this change over here: https://github.com/openshift/origin/pull/28892

Last attempt (first time with the new test) did not install, so learned my lesson and launched three. None the less review would be helpful here.

/hold

File: pkg/monitortests/testframework/metricsendpointdown/monitortest.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	"github.com/openshift/origin/pkg/monitortestframework"
+	"github.com/openshift/origin/pkg/monitortests/clusterversionoperator/operatorstateanalyzer"
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+	"github.com/openshift/origin/pkg/test/ginkgo/junitapi"
+	"k8s.io/client-go/rest"
+)
+
+const testName = "[sig-node] kubelet metrics endpoints should not be unreachable"
```
Comment (neisw) [inline pkg/monitortests/testframework/metricsendpointdown/monitortest.go:18]: 'should not be unreachable' -> 'should be reachable' might be a little more readable.

**Comment by neisw** [comment]
/lgtm

/hold

nit on the test naming but release the hold if you prefer to get it merged as is

File: pkg/monitortests/testframework/metricsendpointdown/monitortest.go (hunk 1 above)
Comment (dgoodwin) [reply pkg/monitortests/testframework/metricsendpointdown/monitortest.go:18]: changed to "should always be reachable"

**Comment by dgoodwin** [comment]
We need to get this back into 4.16 and prior to the test rename, this had passed all required presubmits. As such I think I'm going to override everything so we can get the backport started.

**Comment by neisw** [comment]
The bot really wants you to know that all tests passed...

**Pull request merged**